docs/data-download.md

```diff
@@ -264,7 +264,19 @@ If you are using Binance for example:
 
 ```bash
 mkdir -p user_data/data/binance
-cp freqtrade/tests/testdata/pairs.json user_data/data/binance
+cp tests/testdata/pairs.json user_data/data/binance
```

Comment:
i'm not sure about this in general (not your change per se - but this whole line).

This is intended to only serve as a sample - the pairs in this file may not even exist on binance anymore ... 

I'll think about it ... :laughing: